host/lib/usrp/dboard/e3xx/e3xx_radio_control_impl.cpp

```diff
@@ -263,6 +268,10 @@ void e3xx_radio_control_impl::set_rx_agc(const bool enb, const size_t chan)
     std::lock_guard<std::recursive_mutex> l(_set_lock);
     RFNOC_LOG_TRACE("set_rx_agc(enb=" << enb << ", chan=" << chan << ")");
     const std::string rx_fe = get_which_ad9361_chain(RX_DIRECTION, chan);
+    // Upon enabling AGC, stop tracking by power
```

Comment:
Good catch! I probably would have missed this. If you have a chance, can you maybe use clang-format to format the C++ according to our guidelines.